From 050a8c2de9f3e1f4859abf9b36d2f18afd4c34d7 Mon Sep 17 00:00:00 2001
From: Hs_Yeah <bYeahq@gmail.com>
Date: Tue, 19 Sep 2023 03:12:47 +0800
Subject: [PATCH] Added AmbientCapabilities to nqptp.service.in

Added AmbientCapabilities=CAP_NET_BIND_SERVICE
so that the systemd service can be used without the  capability set on the built nqptp binary.
---
 nqptp.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/nqptp.service.in b/nqptp.service.in
index 6f1eb0c..53e6a2e 100644
--- a/nqptp.service.in
+++ b/nqptp.service.in
@@ -8,6 +8,7 @@ Before=shairport-sync.service
 ExecStart=@prefix@/bin/nqptp
 User=nqptp
 Group=nqptp
+AmbientCapabilities=CAP_NET_BIND_SERVICE
 
 [Install]
 WantedBy=multi-user.target
